docs/source/Technical-Glossary.rst

```diff
@@ -0,0 +1,188 @@
+security between your machine and the virtual one so that yours is not damaged or messed with. Docker eliminates the
+need for the hypervisor because there is no longer a Guest OS. The Docker Engine is software downloaded directly onto
+your machine, and the containers run on the engine. Using Docker eliminates extra steps needed for the VM, as it doesn't
+have to virtualize an entire computer. This makes Docker faster, more efficient, and less redundant than VMs.
+
+.. image:: https://cdnssinc-prod.softserveinc.com/img/blog/containers-security-virtual-machines.PNG
+
+image credits: `docker.com <https.docker.com>`_
+
+For a more in-depth explanation of Docker and how it works, consder looking at `this series of articles.
+<https://www.nschoe.com/articles/2016-05-26-Docker-Taming-the-Beast-Part-1.html>`_
+
+
+Postgres
+--------
+
+PostgreSQL is a general purpose and object-relational database management system. It allows you to store and query large
+amounts of data without having to worry about all of the complicated processes going on under the hood. [*]_  PostgreSQL
+optimizes data querying for you, making your application faster. All information and metadata is stored in it.
+
+
+
+
+RabbitMQ
+---------
+
+RabbitMQ is a message broker. When messages are sent online, they go from the producer to a queue and then to the
+consumer. RabbitMQ is that queue. Instead of having to perform all tasks involving sending messages, including generating
+PDFs, locating the recipient, etc., the message producer only has to upload their message and instructions to the queue
+and RabbitMQ will take care of the rest. Using this service makes messaging through Lookit easier and more efficient, as
+it is able to re-queue messages, it is faster and more reliable, and it is scalable for when there are a lot of messages.
+
+`source <https://www.cloudamqp.com/blog/2015-05-18-part1-rabbitmq-for-beginners-what-is-rabbitmq.html>`_
+
+
+Ngrok
+-----
+
+Ngrok is used in the development process to act as a tunnel into your PC. It is not secure to allow access into your PC
+or local address through a public channel, as this can open you up to malicious attacks. Ngrok allows you to securely
+provide acces to your local address through something public, like the internet. When Ngrok is running on your machine,
+it gets the port of the network service, which is usually a web server, and then connects to the Ngrok cloud service.
+This cloud service takes in traffic and requests from a public address and then relays that traffic directly to the
+Ngrok process running on your machine, which then passes it along to the local address you specify. By doing this, you
+can minimize the interaction between outside traffic and your personal machine.
+
+When trying to stream videos on the development stage, AWS will need an address to send the video to. Ngrok will create
```

Comment:
This is SUPER COOL and a much better workflow for development, I think I had previously only used ngrok to use a local webhook handler for renaming videos but they were still going to S3. @Datamance can you confirm the flow of data here - is AWS still getting the video and then sending it to your computer somehow? Is this for webcam video recording in ember-lookit-frameplayer, videos downloaded by link from GCP, both?

Reply:
Addressed with Mayowa